openstack/nova-hypervisor-agents/templates/_kvm-configmap.yaml.tpl

```diff
@@ -19,4 +19,5 @@ data:
     log_level = 3
     log_outputs = "3:stderr"
     listen_addr = "127.0.0.1"
+    live_migration_inbound_addr = $my_ip
```

Comment:
Sounds unrelated to serial - does "Utilizes $my_ip (host ip) to simplify configuration." mean "unrelated to serial, I also utilized $my_ip" everywhere it fits? If yes, can you do that in its own commit, please, and explain why the previous setting should be changed i.e. what the problems were we encountered?

Reply:
it is unrelated, true. Before that, https://github.com/sapcc/helm-charts/pull/9665/commits/26a684625fc163dd05a6e0f3f8b9482f5865f698#diff-150da47e3e75f3df9c7a003ee26ce9bb25e33382641bbcd5f9138eff501a30b5L114 injected the host ip for the configuration value `live_migration_inbound_addr`, alongside of other configuration flags that need the same value. 

I've refactored the nova-hypervisor-agents a little bit to introduce $my_ip, which is a variable that will hold he ip address of the host but can be reused anytime, thus reduces the amount of injected environment variables to only one.